training/components.py

```diff
@@ -1,5 +1,5 @@
 # type: ignore
-# pylint: disable=import-outside-toplevel,missing-function-docstring
```

Comment:
Why is this being changed?

Reply:
Because we need `import-error` in here due to:
![image](https://github.com/user-attachments/assets/bf7bfb98-6fa5-4a2b-83c1-56646086c931)


And it seems like `missing-function-docstring` is just info severity + we do not declare it anywhere else anyways (except for the one mention I've removed below)...